app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,14 @@
+class LikesController < ApplicationController
+  def create
+    @like = Like.new like_params
+
+    @like.save if @like.valid?
+    redirect_to post_path @like.post
+  end
+
+  private
+
+  def like_params
+    { user: current_user, post_id: params.require(:post_id) }
+  end
```

Comment:
- [ ] There is no need to whitelist the params in this case, you can easily create like:

```ruby
def create
      @post = Post.find(params[:post_id])
      @like = @current_user.likes.new(post_id: @post.id)

      respond_to do |format|
        if @like.save
          format.html { redirect_to @post }
        else
          format.html { redirect_to @post }
        end
      end
    end
```